### ISSUE TITLE
Navigating away from a draft reply message loses the association with the quoted message.

### ISSUE THREAD

**Issue by ianmacd** [oxen-io/session-desktop#2304]
Navigating away from a draft message that quotes another loses the association with the quoted message.

**To Reproduce**

1. Select a conversation.
2. Start a reply to a previously received message, but do not send the new message.
3. Select a different conversation.
4. Select the original conversation.
5. Observe that the previously typed text in the compose box is now a new message, not a reply. The quoted message has been lost.

**Screenshots or Logs**
If applicable, add screenshots or log files to help explain your problem.

**Other information (please complete the following information):**

* Session version 1.8.5.

**Comment by KeeJef** [comment]
Reproducible for me

**Comment by Bilb** [comment]
yeah, we can currently only track a single quoted message accros the whole app.
They are stored in the redux store
`quotedMessageProps`
We would need to add a way to keep track of one for each conversation, in the redux store.
Not so easy unless you already now how redux works.

**Comment by Bilb** [comment]
you can map the same logic as for the `getStagedAttachmentsForConversation`

**Comment by ianmacd** [comment]
> yeah, we can currently only track a single quoted message accros the whole app.

Even if there is only a single message currently being composed across the whole application, and that message quotes another message, when one navigates to another chat and then returns to the chat with the message being composed, the association between that message and the one it was quoting is lost.
